index.js

```diff
@@ -293,33 +293,45 @@ class Plugin extends BtpPlugin {
       await this._writeQueue
     }
 
-    await this._claimFunds()
+    // runs autoclaim in order to ensure that this claim is profitable
+    await this._autoClaim()
 
     clearInterval(this._claimIntervalId)
     debug('done')
   }
 
   async _isClaimProfitable () {
```

Comment:
`_isClaimProfitable` made sense as a function name when it returned a boolean. Now that it returns `profitable` as well, maybe `_getFeeInfo` or similar?